knowledge-base/drawer-add-expand-collapse-handle.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Add Expand/Collapse Handle to toggle the Drawer
+description: How to add Expand/Collapse Handle to toggle the Drawer?
+type: how-to
+page_title: Add Expand/Collapse Handle to toggle the Drawer
+slug: drawer-kb-add-expand-collapse-handle
+position: 
+tags: telerik, blazor, drawer, expand, collapse, handle
+ticketid: 1527830
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Drawer for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+It would be nice to have a handle feature within the rendered Drawer component that would expand and collapse the component.
```

Comment:
Capturing the customer's context does not require copy-pasting their full message. We should only take the keywords and phrases, which they may  use in a Google search.

```suggestion
How to add a handle feature within the rendered Drawer component that would expand and collapse the component?
```